src/main/java/com/elastic/support/diagnostics/DiagnosticInputs.java

```diff
@@ -106,28 +122,35 @@
     public int remotePort = 22;
     // End Input Fields
 
+    String[] typeEntries;
+
+    public DiagnosticInputs(){
+        super();
```

Comment:
Here's the thing though. I added these after when testing my Docker container tests were failing. Because of the output directory not being correctly reset in the BaseInputs constructor. For some reason it wasn't firing. I came back to it the next day and added the super in the DiagnosticInputs and things seemed to be working. So that was a pre-emptive insurance policy to get it running. After revisiting it, I removed the super and it's firing off for some reason. That part of the code doesn't change much, so I'm at kind of a loss on this one.